engine/src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -119,13 +119,16 @@ object Run  {
       } toSeq
     } else Seq.empty
 
+    val acceptableEvents = Set("start", "pulling-image", "localizing-files", "running-docker", "delocalizing-files", "ok", "fail", "start-shutdown")
```

Comment:
This could go up to the top of the `Run` class and become a constant (no need to make a new copy of this for every invocation of this method).